src/cryptoUtils.js

```diff
@@ -35,9 +35,8 @@ export function randomString(size: number): string {
 }
 
 // Returns a new random alphanumeric string suitable for object ID.
-export function newObjectId(): string {
-  //TODO: increase length to better protect against collisions.
```

Comment:
It's funny, I must have seen that comment over a year ago and it never registered that there might be a problem :)